hooks/applypatch-msg

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+# Base Git hook template from https://github.com/rycus86/githooks
+#
+# It allows you to have a .githooks folder per-project that contains
+# its hooks to execute on various Git triggers.
+
+# Read the runner script from the local/global or system config
+# Auto-generated: do not edit!
+GITHOOKS_RUNNER="$(cd "$(dirname "$0")/../" && pwd)/base-template.sh"
```

Comment:
What is the actual file this points at?
Is it ./base-template.sh in the Githooks root?
Should it point to the one in ./release ?

Reply:
it should point too
`base-template.sh` in the githooks root directory (might be a git repo but might not, user may have downloaded a zip version)...
Be aware these hooks are just here to be autogenerated, we do not use them yet, nor do we have tests with which we could tests a simple "clone-and-use" workflow.

Reply:
OK, if we don't use them then I'm OK to ignore it for now, we'll sort out where these need to point to later.